src/api/posts/src/routes/posts.js

```diff
@@ -66,7 +68,7 @@ posts.get('/', validatePostsQuery(), async (req, res) => {
       // Return id and url for a specific post
       .map((id) => ({
         id,
-        url: `/posts/${id}`,
+        url: `${postsUrl}${id}`,
```

Comment:
As a convention, we don't tend to define URLs with a trailing slash.  This means that you need to add it when building pathnames.  This should be:

```js
url: `${postsUrl}/${id}`,
```